src/pkg/exporter/cmd/main.go

```diff
@@ -74,6 +74,19 @@ func forwardEvent(reportData *v1alpha1.ReportData) {
 			enabledOutputs = append(enabledOutputs, "openSearch")
 		}
 	}
+	if config.Governor.URL == "" || config.Governor.ClusterID == "" || config.Governor.CspSecretName == "" {
+		log.Error("Either URL or ClusterID or CSPSecretName is empty")
+		return
+	} else {
+		governorClient, err := outputs.NewClient(
```

Comment:
This is HTTP Client. None of the methods w.r.t this client is not valid for Governor.
We can directly call GovernorPost or initialize Governor Client from here, we no need to map GovernorPost to the same client structure.